### DISCUSSION TITLE
Isar gen import conflict

### DISCUSSION THREAD

**Discussion by tonyhart7** [isar/isar#333] [Help]
isar gen dependency need an update 

incompatible with mocktail
~~~
because vectura depends on both isar_generator ^2.2.1 and mocktail ^0.3.0, version solving failed.)
exit code 1
~~~

**Comment by mstniy** [comment]
This workaround might work: [https://github.com/isar/isar/issues/277#issuecomment-1066093442](https://github.com/isar/isar/issues/277#issuecomment-1066093442)
